scripts/gen-function/parse.ts

```diff
@@ -15,13 +15,16 @@ import { Counter, regexIndexOf } from "./utils.ts";
  * ```
  */
 export function parse(content: string): Definition[] {
+  // Remove modeline
+  content = content.replace(/\n vim:[^\n]*\s*$/, "");
+
   const definitions: Definition[] = [];
   for (const match of content.matchAll(/\*(\w+?)\(\)\*/g)) {
     const fn = match[1];
-    const i = match.index || 0;
+    const i = match.index ?? 0;
     const s = content.lastIndexOf("\n", i);
-    const ms = regexIndexOf(content, /\n[<>\s]/, i);
-    const me = regexIndexOf(content, /\n[^<>\s]/, ms);
+    const ms = regexIndexOf(content, /\n[<>\s]|$/, i);
+    const me = regexIndexOf(content, /\n[^<>\s]|$/, ms);
```

Comment:
`ms` or `me` may resolve to `-1`.
Then the range will be resolved from the beginning of the content.